software/modules/ohdirector/startup.py

```diff
@@ -0,0 +1,23 @@
+import valkyriedriver
+import valkyriedriverpanel
+import exampletaskpanel
+
+from director import tasklaunchpanel
+from director import applogic
+
+def startup(robotSystem, globalsDict=None):
+
+    valkyrieDriver = valkyriedriver.ValkyrieDriver()
+    valkyrieDriverPanel = valkyriedriverpanel.init(valkyrieDriver)
+
+    atlasPanelAction = applogic.getToolBarActions()['ActionAtlasDriverPanel']
+    applogic.getMainWindow().panelToolBar().removeAction(atlasPanelAction)
```

Comment:
this is a sneaky way to remove the Atlas Driver Panel.  Note, the atlas driver and panel are still loaded by startup.py.  In the future, we could have a config file which specifies which drivers/panels to load.  Currently, the atlas driver is used by several other modules to execute behavior commands and other things like that.